Hello Microverse Project

PromzzyKoncepts opened the pull request:
Hi,
I trust you are having a great day!
Here are the features of this pull request:
- HTML file with a heading was created
- CSS file with styling was created
- test.md file with text was created with .gitignore file
- The linters were installed and tested during the project
- The files were added, committed and pushed.

Please review kindly
Thank you.

PromzzyKoncepts committed:
Add html file

PromzzyKoncepts committed:
Add css file

PromzzyKoncepts committed:
Add test.md file

OmarRamoun commented on index.html:
```diff
@@ -0,0 +1,14 @@
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>HELLO MICROVERSE</h1>
```
- [ ] [REQUIRED] Please, make sure to add a class to the `h1` tag as requested in project requirements. And then use that class to add the color to separate style from structure.

PromzzyKoncepts replied:
okay! thanks for the review. Please should i edit in my code editor and commit/push again from same branch  or should i create a new one
how do i resend because i am done with the correction

OmarRamoun reviewed (changes requested):
# 🟡 Almost There! 🏃

## Hi @PromzzyKoncepts 👋

## Welcome to Microverse! 🌎

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights 🌟

- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [![@omarramoun](https://img.shields.io/github/followers/omarramoun?label=Ramoun&style=social)](https://github.com/omarramoun)
- This is My **Twitter**: [![@omarramoun](https://img.shields.io/twitter/follow/omarramoun?label=ramoun16&style=social)](https://twitter.com/omarramoun)
- This is My **Slack**: [![Ramoun](https://img.shields.io/badge/slack-@Ramoun-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

PromzzyKoncepts committed:
Make changes to index.html

PromzzyKoncepts committed:
Add changes made to style.css

PromzzyKoncepts commented:
Hello,
pls observe the changes made to the "Hello Microverse Project"

- I added a class of 'h1' to the h1 tag in HTML
- I added a style to the class h1 in the CSS
- I committed and pushed both to the branch "Code-writing"

youmari reviewed (approved):
# Project Approved :trophy: :tada: 🟢 

Hi @PromzzyKoncepts 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
